src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -518,4 +520,32 @@ def remove_file_post
       end
     end
   end
+
+  describe 'GET #trigger_services' do
+    let(:post_url) { "http://localhost:3200/source/#{source_project}/#{service_package}?cmd=runservice&user=#{user}" }
```

Comment:
I don't know how I can use a regular expression for that, but I could use something like: 

`"http://#{CONFIG['source_host']}:#{CONFIG['source_port']}/source/#{source_project}/#{service_package}?cmd=runservice&user=#{user}"`

Is that fine?